### ISSUE TITLE
Indicator that tab will close on next "back".

### ISSUE THREAD

**Issue by ElhemEnohpi** [bomjacob/back2close#9]
Because the back button always shows that there's a previous page, it would be nice if there was a toolbar button or some other indicator to tell you that the tab will close the next time you go "back". I often accidentally close a tab because I think there's still a page before it.

Oh, l just saw that someone already asked for this in a review. The downsides you mentioned wouldn't be problem for me. It doesn't need to replace the Back button, just a small indicator I could put beside it would be fine...

**Comment by jsmnbom** [comment]
Hmm, that could possibly be implemented... I would have to look into how feasible it is to have such an indicator be toggleable, since I assume not everyone would want it.

**Comment by ElhemEnohpi** [comment]
I assumed it would be a normal toolbar button, that people could optionally put next to the back button (or wherever they want). The button would change its appearance when the page is the last one (preferably not only by changing colour, because I use an addon that removes colour from the buttons).

**Comment by jsmnbom** [comment]
This should be nicely possible, when mozilla allows themes to change icons...
Currently it's behind a perf https://bugzilla.mozilla.org/show_bug.cgi?id=1348039.
When that perf is enabled by default, I will add the theme permission, and change the icon when the last history entry will close the tab, but until then, I'd rather not add another permission.

**Comment by emvaized** [comment]
It would be a nice feature, but the proposed toolbar button solution would not work for me, as I usually use mouse back/forward buttons for navigating pages. 

Maybe showing default system notification with [chrome.notifications ](https://developer.chrome.com/docs/extensions/reference/notifications/) API would be ok? Some people for sure may find it annoying, but the option could be disabled by default.

**Comment by CennoxX** [comment]
Uh, I can't imagine there is anyone, who wouldn't be annoyed by this.

**Comment by emvaized** [comment]
> Uh, I can't imagine there is anyone, who wouldn't be annoyed by this.

Yeah, notifications are annoying, but at least they should solve the problem, while being easy to implement and disabled by default. 

Honestly, I probably would be satisfied with just a blacklist option for excluded domains, because most of the time I accidentally close only Github tabs

**Comment by CennoxX** [comment]
> It would be a nice feature, but the proposed toolbar button solution would not work for me, as I usually use mouse back/forward buttons for navigating pages.

Why wouldn't this work for you? Even if you use the mouse buttons, you could notice a color change on a toolbar icon, or what would be your the problem with that?

**Comment by emvaized** [comment]
Because it could be too far from my eyes focus point on the page when I decide to perform the "Go back" acrion using the mouse, due to the fact my monitor is quite big
